## /bin/sh -e simulate dpatch for prepend_chlog.py
## Build
## less_docs.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
## DP: As noone really uses most of the API documentation, let's leave
## DP: just the important bits

Index: asterisk-1.2.24.dfsg+pf.0.1~svn1329/contrib/asterisk-ng-doxygen
===================================================================
--- asterisk-1.2.24.dfsg+pf.0.1~svn1329.orig/contrib/asterisk-ng-doxygen	2007-08-08 14:42:39.000000000 +0200
+++ asterisk-1.2.24.dfsg+pf.0.1~svn1329/contrib/asterisk-ng-doxygen	2007-08-08 14:43:15.000000000 +0200
@@ -214,7 +214,7 @@
 # Private class members and static file members will be hidden unless 
 # the EXTRACT_PRIVATE and EXTRACT_STATIC tags are set to YES
 
-EXTRACT_ALL            = YES
+EXTRACT_ALL            = NO
 
 # If the EXTRACT_PRIVATE tag is set to YES all private members of a class 
 # will be included in the documentation.
@@ -224,13 +224,13 @@
 # If the EXTRACT_STATIC tag is set to YES all static members of a file 
 # will be included in the documentation.
 
-EXTRACT_STATIC         = YES
+EXTRACT_STATIC         = NO
 
 # If the EXTRACT_LOCAL_CLASSES tag is set to YES classes (and structs) 
 # defined locally in source files will be included in the documentation. 
 # If set to NO only classes defined in header files are included.
 
-EXTRACT_LOCAL_CLASSES  = YES
+EXTRACT_LOCAL_CLASSES  = NO
 
 # This flag is only useful for Objective-C code. When set to YES local 
 # methods, which are defined in the implementation section but not in 
@@ -392,7 +392,7 @@
 # The QUIET tag can be used to turn on/off the messages that are generated 
 # by doxygen. Possible values are YES and NO. If left blank NO is used.
 
-QUIET                  = NO
+QUIET                  = YES
 
 # The WARNINGS tag can be used to turn on/off the warning messages that are 
 # generated by doxygen. Possible values are YES and NO. If left blank 
@@ -445,18 +445,9 @@
 # directories like "/usr/src/myproject". Separate the files or directories 
 # with spaces.
 
-INPUT                  = ./ \
+INPUT                  = \
 			 include \
-                         include/asterisk \
-			 channels \
-			 funcs \
-			 stdtime \
-			 apps \
-			 cdr \
-			 codecs \
-			 formats \
-			 pbx \
-			 res
+                         include/asterisk 
 
 # If the value of the INPUT tag contains directories, you can use the 
 # FILE_PATTERNS tag to specify one or more wildcard pattern (like *.cpp 
